SConstruct

```diff
@@ -355,6 +358,61 @@ if GetOption("lang-python"):
 
     Default(swiginstall, copySwig)
 
+# Javap SWIG wrapper for WiredTiger
+enableJava = GetOption("lang-java")
+if enableJava and "," in enableJava:
+    enableJavaPaths = enableJava.split(',')
+    if len(enableJavaPaths) == 2:
+        swigExe = enableJavaPaths[0]
```

Comment:
@frasator, I don't think this catches the error with `lang-java=x,y,z`, won't that be silently accepted?
Other than that, I like the changes you've made.